smart-table-module/js/Table.js

```diff
@@ -223,7 +227,7 @@
                 var i = 0,
                     l = scope.displayedCollection.length;
 
-                if (scope.selectionMode !== 'multiple') {
+                if (scope.selectionMode !== 'multiple' && scope.selectionMode !== 'customMultiple') {
```

Comment:
IMHO, it's better to have a function to determine if the mode is in multiple or not. 
function isMultiple() {
 return scope.selectionMode === 'multiple' || scope.selectionMode === 'customMultiple'
}